src/test/a11y/a11y.ts

```diff
@@ -68,7 +67,7 @@ function ensureHeadingIsIncludedInPageTitle (text: string): void {
     expect(title).to.be.equal(`${heading[1]} - Money Claims`)
   } else {
     expect(title).to.be.not.equal(' - Money Claims')
-    logger.warn(`No heading found on page titled '${title}' exists`)
+    console.log(`NOTE: No heading found on page titled '${title}' exists`)
```

Comment:
do we want to use `console.log`?

Reply:
Yes, it gives me better results then when I used logger. I am OK with console.log to print output in tests.